accord-core/src/main/java/accord/coordinate/Recover.java

```diff
@@ -262,7 +261,12 @@ private void recover()
                 case Applied:
                 case PreApplied:
                 {
-                    withCommittedDeps(executeAt, stableDeps -> {
+                    withCommittedDeps(executeAt, (stableDeps, failure) -> {
+                        if (failure != null)
+                        {
+                            // TODO (review): This case we don't pass a callback to persist, is that intentional?
```

Comment:
so its unhandled?  so shouldn't the `agent` be notified then?

Reply:
I reworked this to have them all pass a wrapped exception to the agent.